tests/test_convertors.py

```diff
@@ -1,15 +1,20 @@
 from datetime import datetime
+from typing import Any, Callable, Generator
 
 import pytest
 
 from starlette import convertors
 from starlette.convertors import Convertor, register_url_convertor
+from starlette.requests import Request
 from starlette.responses import JSONResponse
 from starlette.routing import Route, Router
+from starlette.testclient import TestClient
+
+TestClientFactory = Callable[..., TestClient]
 
 
 @pytest.fixture(scope="module", autouse=True)
-def refresh_convertor_types():
+def refresh_convertor_types() -> Generator[Any, None, None]:
```

Comment:
It's not `Any`, it's None. But we can use `Iterator` here. 
```suggestion
def refresh_convertor_types() -> Iterator[None]:
```